install-instructions.md

```diff
@@ -119,17 +119,73 @@ For macOS or Linux [you need Java](https://adoptium.net/) to run the Fabric Inst
+
+<details>
+  <summary>I agree and would like to see the instructions</summary>
+
+1. [Download Pojav Launcher](https://play.google.com/store/apps/details?id=net.kdt.pojavlaunch)
+   * Do not fall into scam websites and apps. The only official site for it is https://github.com/PojavLauncherTeam/PojavLauncher
+2. [Download Files](https://play.google.com/store/apps/details?id=com.marc.files)
+   * This is a shortcut to your system's file manager. Other file managers may need root to follow these instructions.
+3. [Download Fabric installer](https://fabricmc.net/use/installer/) (Universal/.JAR)
+4. [Download the latest **MultiMC version** of Fabulously Optimized](https://www.curseforge.com/minecraft/modpacks/fabulously-optimized/files)
+   * On CurseForge's new UI, `⌵` → `Download file`
+   * Don't see the MultiMC version? Click the title of the CurseForge version you need and scroll down, you'll find the MultiMC variant there.
+      * Alternatively, look for it in the sidebar, under "server packs". Don't worry, these are not server packs, just marked as such for findability.
+   * If you don't see any mods inside the zip or only see one, you downloaded the wrong version.
+5. Download Fabulously Optimized patches for Pojav Launcher: [1.19.3](https://minhaskamal.github.io/DownGit/#/home?url=https://github.com/Fabulously-Optimized/fabulously-optimized/tree/pojav/PojavLauncher/1.19.3)
+   * This optimizes the modpack for mobile users.
+6. Run Pojav Launcher
+7. Tap `Add account` → `Microsoft account` and sign in
+8. Tap `(Default) - Unknown` → `Create new profile` → `SAVE`
+9. Tap `Play`. Wait for it to fully load to the title screen, then exit the game.
+    * If you get a RAM warning ("The current amount of..."), tap `OK` for now. It usually means you just need to close background apps for better performance.
+    * If you get a black screen: quit the game, go back to Pojav → `🎛️` → `Video and renderer` → `Renderer` → (choose any) → ⬅️ → ⬅️ → try playing again
+10. Launch Pojav again, tap `Install .jar` → select `fabric-installer-x.x.x.jar`. You'll see the Fabric installer with some logs.
+11. Tap `❌` to close the logs, then `Install` → `OK` → `⛝`
+12. Close Pojav and open Files.
+13. Tap `☰` → `PojavLauncher` → hold down on `options.txt`, tap `🗑️` → `OK`
+14. Tap `☰` → `Downloads` → `Fabulously+Optimized+x.x.x.zip` → `Fabulously Optimized x.x.x` → `minecraft`
+15. Hold down on one folder and tap on all three folders, then tap `⋮` → `Extract to...`
+16. Tap `☰` → `PojavLauncher` → `⋮` → `Show hidden files` → `.minecraft` → `EXTRACT`
+17. Tap `☰` → `Downloads` → `DownGit.zip` → `(Minecraft version)`
+    * Can't see .minecraft? Tap `⋮` → `Show hidden files` 
+18. Hold down on `config` folder and tap on both items, then tap `⋮` → `Extract to...`
+19. Tap `☰` → `PojavLauncher` → `.minecraft` → `EXTRACT`
+20. Tap `☰` → `PojavLauncher` → `.minecraft` 
+21. On each duplicated folder (e.g. "config (2)" or "resourcepacks (2)"), do the following:
```

Comment:
This is so stupid, but it seems to be what the default file manager does so I'm not sure how to workaround it well. Using a different file manager doesn't seem to be an option after all because

1. it makes Pojav files much harder to access
2. it may require root to access them in the first place